gov/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/decred/dcrdata/gov/v4
+module github.com/decred/dcrdata/gov/v5
```

Comment:
Oh dang actually we never reached a gov/v4 release (no gov/v4.x.y tag exists).  Sorry, can you go back to gov/v4 since we never closed dev cycle that would have stabilized it?  My bad.

Reply:
no problem at all mate, all good 👍 rewinded the package version changes